clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -78,3 +78,15 @@ endmacro()
 # Always set SYCL_HAS_FP16 to true to assume SYCL device to support float16
 message("dpct.cmake: SYCL_HAS_FP16 is set true by default.")
 set(SYCL_HAS_FP16 TRUE)
+
+# Replaces cuda_compile with it's equivalent SYCL repr
+macro(DPCT_COMPILE generated_files)
+  DPCT_GET_SOURCES(_sources ${ARGN})
+
+  # can't continue without list of source files
+  if("${_sources}" STREQUAL "")
+    message(FATAL "dpct: failed to find source files while migrating `cuda_compile`")
```

Comment:
It's build time that the message in Line88 will be print out (not during migration),  so how about update the message to "Failed to find source files when run DPCT_HELPER_SYCL_COMPILE macro".

Reply:
Update the message to following
> Failed to find the source files while running the macro 'DPCT_HELPER_SYCL_COMPILE'